### PR TITLE
change bazel-mirror to mirror.bazel

### PR DESCRIPTION
**Pull request by eigen2017**
hi i'm back , using only one email.

### CODE REVIEW

**Comment by eigen2017** [comment]
cla signe issue seems solved

**Comment by eigen2017** [comment]
here's the path for passing cla:
1.use google account, say wobushiliu2@gmail.com to register github, 
   and set a git name, say eigen2017.

2.goto https://cla.developers.google.com/clas to sign a agreement as "only self".
   set "GitHub Account Name" to be wobushiliu2@gmail.com

3.jus fork a copy of tensor project and clone it to local:
➜  tensorflow git:(master) git remote -v
origin	https://github.com/eigen2017/tensorflow (fetch)
origin	https://github.com/eigen2017/tensorflow (push)

4.set name and email:
git config user.name "eigen2017"
git config user.email "wobushiliu2@gmail.com"
be care not to add "global" option, so the git configuration is only for ur fork project , not for other git project on your pc.

5.change the source code

6. git add *
    git commit -m "..."
    git push origin master

7. grab a pull req from https://github.com/tensorflow/tensorflow/pulls
    select:  "compare across forks"
    select: your fork 

8. post the pull req

**Comment by eigen2017** [comment]
@jart  i was jakiechris, u can call me eigan now  :)

**Review by jart — APPROVED** [review]

**Comment by jart** [comment]
Welcome back! Thank you for the contribution. Mr. Jenkins: test this please.

**Comment by sb2nov** [comment]
@eigen2017 can you look into the merge conflict

**Comment by sb2nov** [comment]
@eigen2017 I might have made a mistake in resolving commits via the web UI. Can you just fix it and force push. Sorry about that.

File: tensorflow/contrib/makefile/download_dependencies.sh [hunk 1]
```diff
@@ -26,13 +26,13 @@ if [ ! -f $BZL_FILE_PATH ]; then
...
 
-EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v bazel-mirror | head -n1)"
+EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v mirror.bazel | head -n1)"
 GEMMLOWP_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/gemmlowp/.*zip' "${BZL_FILE_PATH}" | head -n1)"
 GOOGLETEST_URL="https://github.com/google/googletest/archive/release-1.8.0.tar.gz"
 NSYNC_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/nsync/.*tar\.gz' "${BZL_FILE_PATH}" | head -n1)"
 PROTOBUF_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/protobuf/.*tar\.gz' "${BZL_FILE_PATH}" | head -n1)"
-RE2_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/re2/.*tar\.gz' "${BZL_FILE_PATH}" | head -n1)"
-FFT2D_URL="$(grep -o 'http.*fft\.tgz' "${BZL_FILE_PATH}" | grep -v bazel-mirror | head -n1)"
+RE2_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/re2/.*tar\.gz' "${BZL_FILE_PATH}" | head
```
Comment (jart) [inline tensorflow/contrib/makefile/download_dependencies.sh]: Why was the end of this line removed?

**Review by jart — CHANGES_REQUESTED** [review]
This

File: tensorflow/contrib/makefile/download_dependencies.sh (hunk 1 above)
Comment (sb2nov) [reply tensorflow/contrib/makefile/download_dependencies.sh]: I resolved it incorrectly while trying to use the WebUI. I'll fix that.

**Comment by jart** [comment]
I'm closing in favor of https://github.com/tensorflow/tensorflow/pull/15007

**Comment by jart** [comment]
Actually I'm re-opening. I didn't realize that the second person who committed to this branch was the TF sync on-call. My mistake.

**Review by jart — APPROVED** [review]

**Comment by sb2nov** [comment]
Let's merge #15007 as the email address used in the commit is some private email created by github due to my email being hidden.

**Comment by jart** [comment]
Sounds good. In that case, I'm sorry this change didn't work out @eigen2017. Sometimes sync causes headaches.

**Comment by eigen2017** [comment]
ok, next time i'll synchronize the codes before changing it.

**Pull request closed without merging**